docs/getting-started-guides/README.md

```diff
@@ -6,9 +6,9 @@ For the easiest "kick the tires" experience, please try the [local docker](docke
 If you are considering contributing a new guide, please read the
 [guidelines](../../docs/devel/writing-a-getting-started-guide.md).
 
-IaaS Provider        | Config. Mgmt | OS     | Networking  | Docs                                                                           | Conformance | Support Level                | Notes
+IaaS Provider        | Config. Mgmt | OS     | Networking  | Docs                                                                           | Conforms    | Support Level                | Notes
 -------------------- | ------------ | ------ | ----------  | ------------------------------------------------------------------------------ | ----------- | ---------------------------- | -----
-GKE                  |              |        | GCE         | [docs](https://cloud.google.com/container-engine)                              |             | Commercial                   | Uses K8s version 0.15.0
+GKE                  |              |        | GCE         | [docs](https://cloud.google.com/container-engine)                              | [✓](1)      | Commercial                   | Uses K8s version 0.15.0
```

Comment:
Oh, I guess only reason 2b is valid: so that the text version of the md has reasonable line length.